scripts/irods/test/test_iticket.py

```diff
@@ -874,3 +874,110 @@ def get_modification_and_creation_time():
         previous_ts = modify_ts
         modify_ts, _ = get_modification_and_creation_time()
         self.assertGreater(modify_ts, previous_ts)
+
+    def test_tickets_update_catalog_modify_time_when_modified_user__issue_7125(self):
```

Comment:
How do you feel about the following?
```
test_ticket_mtime_is_updated_when_ticket_properties_are_modified_by_non_admins__issue_7125
```

Reply:
This looks clearer that what I had. I will implement it.